elasticsearch_dsl/field.py

```diff
@@ -282,39 +287,74 @@ def clean(self, data):
             raise ValidationException("Value required for this field.")
         return data
 
+
 class Float(Field):
     name = 'float'
 
-class HalfFloat(Field):
+    def _deserialize(self, data):
```

Comment:
we need `_coerce = True` here, otherwise the value won't be deserialized properly